.github/workflows/main.yml

```diff
@@ -16,13 +16,14 @@ jobs:
 
       - uses: actions/checkout@v4
 
-      - uses: actions/setup-python@v4
+      - uses: actions/setup-python@v5
         with:
           python-version: "3.11"
-          cache: pip
-          cache-dependency-path: requirements-dev.txt
 
-      - run: pip install -r requirements-dev.txt
+      - name: download uv
+        run: curl -LsSf https://astral.sh/uv/install.sh | sh
+
+      - run: uv pip install --system -r requirements-dev.txt
```

Comment:
It's a little weird that we'll install `uv` here since it's in the dev requirements. Should we add it to the unsafe packages or omit it from the pyproject.toml and add install instructions to a CONTRIBUTING file?

Reply:
I think adding it to the unsafe list is probably the best way to go about it. Because ultimately, the package management used for local dev doesn't really matter. And adding a `CONTRIBUTING` file just for such an inconsequential thing feels excessive.

Reply:
I added `uv` to the emit list. If you feel that this repo now requires a `CONTRIBUTING` file, please let me know or add it. Otherwise, I think this it gtg